tasks/send-pan/tests/index.js

```diff
@@ -45,21 +46,18 @@ test('SendPan task calls upload', async (t) => {
     },
   };
 
-  const url = `http://${event.config.provider.host}`;
-  const remotePath = path.join('/', event.config.remoteDir, `${fileNameBase}.pan`);
+  const url = `http://${event.config.provider.host}:${port}`;
+  const remotePath = path.join(event.config.remoteDir, `${fileNameBase}.pan`);
   // Message should look like this:
   // MESSAGE_TYPE = "SHORTPAN";
   // DISPOSITION = "SUCCESSFUL";
   // TIME_STAMP = 2023-03-27T18:10:56.402Z;
-  nock(url).post(remotePath, regex)
```

Comment:
is this validation happening somewhere else?

Reply:
Yes, and no.   This is one of those cases where we're re-validating *library* behavior that itself mocks.  

got is utterly broken in node20 with respect to nock/aws introspection used with file streams - and other options (axios) don't handle file streaming.    As such, this test is *only* testing that something attempts to post to the nock'd HTTP endpoint, which *is* unit tested in the ingest module already via the workaround noted in `test-HttpProviderClient`.   

Opinion: 
In light of the issues with nock/etc, got and node 20 the value here of requiring this is at least somewhat debatable (or at least pushes the bounds of social/unsocial units) given the code being covered by the unit isn't in this task,  and we have coverage re: a PAN being sent.     Removing this assertion doesn't directly reduce code coverage.   

The alternate approaches include:

1)  Update the HTTP server with a CGI script to actually take any post and write it to the container file system/serve it to validate *something is posted* and evaluate that response in the unit
2) Update the existing integration test to use a HTTP provider.   This would also require an HTTP server that will take POST/put 
3) Exposing the test mock params through the lambda client to this unit test
4) Fix got, or implement an alternative to utilize our test mocking structure.